build.sbt

```diff
@@ -66,3 +66,5 @@ buildInfoOptions += BuildInfoOption.ToMap
 releaseProcess := releaseProcess.value.filterNot(_ == ReleaseTransformations.publishArtifacts)
 
 enablePlugins(sbtbuildinfo.BuildInfoPlugin)
+
+assembly/assemblyJarName := s"${name.value}-assembly.jar"
```

Comment:
With sbtWrapper, we no longer have the option to rename the compilation artifact. With this addition, sbt will give it the name expected by the Dockerfile.